src/racing/actors.c

```diff
@@ -2517,121 +1426,121 @@ s16 addActorToEmptySlot(Vec3f pos, Vec3s rot, Vec3f velocity, s16 actorType) {
     return -1;
 }
 
-s16 func_8029ED38(Vec3f pos, s16 actorType) {
+s16 allocate_actor_with_new_position(Vec3f pos, s16 actorType) {
     Vec3f vel;
     Vec3s rot;
 
     vec3f_set(vel, 0.0f, 0.0f, 0.0f);
     vec3s_set(rot, 0, 0, 0);
-    return addActorToEmptySlot(pos, rot, vel, actorType);
+    return add_actor_to_empty_slot(pos, rot, vel, actorType);
 }
 
-// I swear we have a struct that looks like this already but I cannot find it anywhere
+// not ActorSpawnData but very similar in structure and use
 struct test {
     Vec3s thing;
 };
 
-UNUSED void func_8029ED98(Player *player, uintptr_t arg1) {
+UNUSED void proto_loading_actor_spawn_data(Player *player, uintptr_t arg1) {
```

Comment:
Took me a bit to understand what the name meant.

Since `function prototype` is a term in C. I wonder if it would be confused with `prototype` (I did, but I might be the only one to confuse that).

`prototype_actor_spawn_data`

Reply:
![image](https://github.com/n64decomp/mk64/assets/59367621/2e0d65c0-3e5c-4541-92a2-db37d2aeac61)
I use it like primitive function